### DISCUSSION TITLE
How to Form Setting(margin)

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#3926] [Help] [answered]
Hi All,

How I can utilize the margin ?(showed as red arrow).
I wan make my form more bigger.

Thank You.

![image](https://user-images.githubusercontent.com/6214964/189415323-20e14ea2-4c51-4cc7-8e88-94580aad0d02.png)

**Comment by danharrin** [comment, marked as answer]
Either change the `filament/layout.max_content_width` config to `full`, or just on that page, set `protected ?string $maxContentWidth = 'full';`

**Reply by jigujigu** [reply]
Thank you.
